### ISSUE TITLE
wot no grades?

### ISSUE THREAD

**Issue by thepurpleblob** [davidpesce/moodle-mod_tincanlaunch#135]
This activity (for me anyway) doesn't have any grade options nor does it create an entry in the Gradebook. Is this correct?

What happens to any final grade returned from the tincan object?

**Comment by garemoko** [comment]
That's correct. Please feel free to make a pull request to add this functionality!

**Comment by caperneoignis** [comment]
This would be hard to implement due to the way the content may send back the grade. take for instance, articulate story line, sends the question and response and whether it was correct or not. Then it says if the user passed each quiz, if there are multiple, so the actual passed event will say how many correct out of how many asked, but not really a grade. I have noticed the different content creator software all seem to do the actual events differently, if only slightly, so I'm not sure how this could be implemented with out a case statement for every case one might see. This is all dependent on the actual content returning a grade in the first place, it may only respond with completed, and the grade buried in there. So this will rely heavily on the content being created correctly, and if the content itself sends the data uniformly.

So, in this case, I have moved the quizzes to moodle quizzes and use the xAPI logstore plugin, and use the LRS reporting function to figure out questions and answers stats, and use Moodle to track the grade.
